project/Dependencies.scala

```diff
@@ -102,6 +102,8 @@ object Dependencies {
       namespace %% "interop-be-agreement-management-models" % agreementManagementVersion
     lazy val purposeManagementModels =
       namespace %% "interop-be-purpose-management-models" % purposeManagementVersion
+    lazy val authManagementModels =
```

Comment:
Characters are still free 🤣 
```suggestion
    lazy val authorizationManagementModels =
```